src/coreclr/jit/gentree.cpp

```diff
@@ -19669,6 +19669,8 @@ bool GenTree::isContainableHWIntrinsic() const
         case NI_AVX512F_ConvertToVector256UInt32:
         case NI_AVX512F_VL_ConvertToVector128UInt32:
         case NI_AVX512F_VL_ConvertToVector128UInt32WithSaturation:
+        case NI_AVX10v1_ConvertToVector128UInt32:
```

Comment:
I've not looked at all of them, but we should double check the `compIsaSupportedDebugOnly(InstructionSet_AVX512*)` and `compOpportunisticallyDependsOn(InstructionSet_AVX512*)` and `compExactlyDependsOn(InstructionSet_AVX512*)` checks in this file all make sense, since I found some in other files that looked to not be accounting for Avx10v1 where relevant.

Reply:
true. I will try to look for all such checks and handle ```Avx10v1``` in those. I will update here when I am done with my changes.

Reply:
Hi Tanner,

So the main places where we need to handle this is in the special intrinsic. Correct me if I am wrong. The only place where we have not checked for EVEX support are for special intrinsic. I am waiting to resolve https://github.com/dotnet/runtime/pull/101938#discussion_r1605553081 to add these checks. In the mean time, please do let me know if you think there are other places as well where we need to check for EVEX support. i have left out all the places where simdSize is 64 because that would ensure we do have AVX512F.